### PR TITLE
Release scripts

### PR DESCRIPTION
**Pull request by aeris**
Automate binaries generation and release publication

### CODE REVIEW

File: scripts/release.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+git = Git.open root_dir
+tag = git.describe '--tags'
+
+token = ENV.fetch 'GITHUB_RELEASE_TOKEN'
+github = Octokit::Client.new access_token: token
+user = github.user
+user.login
+
+# releases = github.releases REPO
+# releases.each { |r| github.delete_release r.url }
```
Comment (nono) [inline scripts/release.rb]: Is this commented code for when an upload failed and you want to recreate the release?

File: scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -xe
+
+RELEASE="$(git describe --tags)"
+
+go get -u -v ./... || true
```
Comment (nono) [inline scripts/release.sh]: Why the `|| true`?

File: scripts/release.rb (hunk 1 above)
Comment (aeris) [reply scripts/release.rb]: For testing purpose.
Very tedious to have to manually destroy release to be able to rerun the script without failure…
Can be removed

File: scripts/release.sh (hunk 2 above)
Comment (aeris) [reply scripts/release.sh]: "go get" sometime returns error return code even when no real error, and because of the "set -xe", fails all the script :cry:

**Pull request merged**